bin/stampede-worker.js

```diff
@@ -39,7 +39,7 @@ const conf = require("rc-house")("stampede", {
   environmentVariablePrefix: "STAMP_",
   shell: "/bin/bash",
   gitClone: "https",
-  gitCloneOptions: "",
```

Comment:
Can you also revert this change? Given the overall change to make clone options completely configurable, having a default doesn't make sense. All you need is one config setting for your workers to add the `--filter=tree:0` to each node. But it would be counter-intuitive to make others clear out the options for the case where they wanted to do a full clone. And in fact, leaving the default at `""` feels more correct as that will do the full clone, which should work in all circumstances but just be slow. So this clone options param can just be thought of how to make the clones more efficient. Eventually I think this should be configurable per-task, but that is beyond the scope of this PR and would require changes on the server end as well. Thanks!

Reply:
resolved